Resources/public/js/views/ez-contenteditview.js

```diff
@@ -50,13 +51,35 @@ YUI.add('ez-contenteditview', function (Y) {
             this.after('activeChange', function (e) {
                 if ( e.newVal ) {
                     this._setFocus();
+                    this._removeInfos();
                 }
             });
 
+
+
             this.on(['*:saveAction', '*:publishAction'], this._handleSavePublish);
         },
 
         /**
+         * Hide infos (description and technical infos) when clicking outside the header
+         *
+         * @method _removeInfos
+         * @protected
+         * @param {Object} e event facade
+         */
+        _removeInfos: function (e) {
```

Comment:
this is a very bad name (and description in the comment). This method does not remove anything, it sets an event handler on the `clickoutside` event that will hide the infos. So please rename it to something like `_addClickoutsideInfoHandler` and use `_hideDetails` instead of reimplementing it in an anonymous function.